src/build/optimize.ts

```diff
@@ -26,31 +26,30 @@ export interface OptimizeOptions {
    * Options passed to gulp-html-minifier
    */
   html?: {
-    collapseWhitespace?: boolean;
-    removeComments?: boolean;
+    [property: string]: any;
   };
   /**
    * Enable CSS minification
    */
-  css?: {
-    stripWhitespace?: boolean;
+  css?: boolean | {
+    [property: string]: any;
```

Comment:
The properties in there aren't even used, so I'm not 100% sure what the interface should be anymore. That should probably not be in _this_ PR though.